File: DevExpress-Demo/CS/SpellCheckerDemo.Wpf/SpellCheckerDemoModuleControl.cs
using System.Collections.Generic;
using System.Windows;
using DevExpress.Xpf.DemoBase;
using DevExpress.XtraSpellChecker.Native;
using DevExpress.Xpf.RichEdit;
using DevExpress.XtraRichEdit.SpellChecker;

namespace SpellCheckerDemo {
    public class SpellCheckerDemoModule : DemoModule {
        
        protected virtual List<FrameworkElement> CheckingElements { get { return null; } }

        protected override void ShowPopupContent() {
            base.ShowPopupContent();
            if(CheckingElements == null)
                return;
            foreach(FrameworkElement element in CheckingElements)
                element.Visibility = Visibility.Visible;
        }
        protected override void HidePopupContent() {
            if(CheckingElements == null)
                return;
            foreach(FrameworkElement element in CheckingElements)
                element.Visibility = Visibility.Hidden;
            base.HidePopupContent();
        }
    }
}

File: DevExpress-Demo/VB/SpellCheckerDemo.Wpf/SpellCheckerDemoModuleControl.vb
Imports System.Collections.Generic
Imports System.Windows
Imports DevExpress.Xpf.DemoBase

Namespace SpellCheckerDemo

    Public Class SpellCheckerDemoModule
        Inherits DemoModule

        Protected Overridable ReadOnly Property CheckingElements As List(Of FrameworkElement)
            Get
                Return Nothing
            End Get
        End Property

        Protected Overrides Sub ShowPopupContent()
            MyBase.ShowPopupContent()
            If CheckingElements Is Nothing Then Return
            For Each element As FrameworkElement In CheckingElements
                element.Visibility = Visibility.Visible
            Next
        End Sub

        Protected Overrides Sub HidePopupContent()
            If CheckingElements Is Nothing Then Return
            For Each element As FrameworkElement In CheckingElements
                element.Visibility = Visibility.Hidden
            Next

            MyBase.HidePopupContent()
        End Sub
    End Class
End Namespace
